connectors/http/http-base/src/main/java/io/camunda/connector/http/base/model/HttpCommonRequest.java

```diff
@@ -111,6 +111,13 @@ public class HttpCommonRequest {
       description = "Store the response as a document in the document store")
   private boolean storeResponse;
 
+  @TemplateProperty(
```

Comment:
Would be great to add a label & description to allow users to discover what this new property is all about.